### DISCUSSION TITLE
Customize generated types

### DISCUSSION THREAD

**Discussion by FelixStumvoll** [dotansimha/graphql-code-generator#5995] [Q&A] [answered]
I am currently working on a graphql project and i wanted to try out this library. However i have one issue i can't quite wrap my head around. 

If i have the following type definition, the library will generate a type with exactly these types. However i might want to resolve `jobs` using a resolver. 
```graphql
type Pipeline {
    id: ID!
    name: String!
    jobs: [Job!]!
}
```
I know i can define a custom type using `mappers` in the config. I just wanted to know if this is the best solution to solve this problem

**Comment by dotansimha** [comment, marked as answer]
Yeah, `mappers` is the right way to go with this one :)
